openupgradelib/openupgrade.py

```diff
@@ -64,6 +64,15 @@ def __exit__(self, exc_type, exc_value, traceback):
 else:
     import release
 
+try:
+    from odoo.modules.migration import VALID_MIGRATE_PARAMS
```

Comment:
Can you document this was introduced in 18?

Reply:
thanks, done